manifest.json

```diff
@@ -365,8 +372,8 @@
         {
           "identifier": "show_follow_topic",
           "type": "checkbox",
-          "description": "follow_topic_description",
-          "label": "follow_topic_label",
+          "description": "show_follow_topic_description",
```

Comment:
Not sure I understand this change. Would you like to update these strings as well? If so, both `description` and `label` values should match keys in [translations.yml](https://github.com/zendesk/copenhagen_theme/blob/master/translations.yml#L344).